tools/system_libs.py

```diff
@@ -41,7 +41,7 @@ def read_symbols(path, exclude=None):
       symbols = filter(lambda symbol: symbol not in exclude, symbols)
     return set(symbols)
 
-  default_opts = []
+  default_opts = ['-Werror']
```

Comment:
Hello, may I ask why this is added? It looks like this causes the system library compilation at startup time to break.